db/migrate/20210621160955_change_early_career_teacher_profile_to_make_cohorts_required.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ChangeEarlyCareerTeacherProfileToMakeCohortsRequired < ActiveRecord::Migration[6.1]
+  def change
+    change_column_null :early_career_teacher_profiles, :cohort_id, false, Cohort.current.id
```

Comment:
Setting `NOT NULL` on an existing column isn't safe. Try something like:

```suggestion
    add_check_constraint :early_career_teacher_profiles, "cohort_id IS NOT NULL", name: "early_career_teacher_profiles_cohort_id_null", validate: false
```

Reply:
We would need to define an up and down to make this reversible, right?

Reply:
Ah, it's reversible. Nice!

Reply:
Yes. Was just about to comment on that with:
https://blog.saeloun.com/2021/01/08/rails-6-check-constraints-database-migrations.html